script/subtitles/imscsubtitles.js

```diff
@@ -138,10 +219,33 @@ define('bigscreenplayer/subtitles/imscsubtitles',
         }
       }
 
+      function modifyStyling (xml) {
+        if (liveSubtitles) {
+          xml.head.styling.initials = defaultStyleOpts.initials;
```

Comment:
Is this not protected by the try catch in loading of segments?

Reply:
I guess it depends what we want to do in the situation that the values don't exist. Do we want to no show subs, or show subs in an unstyled manner.

Reply:
So this will currently show subs without this styling override. As part of future work we could emit another event through the plugins so that clients could monitor errors in this case.